src/app/@ansyn/ol/plugins/plugins.config.ts

```diff
@@ -1,7 +1,23 @@
+import { IKeyValuePair } from "./annotations/annotations-context-menu/models/key-value.interface";
+
 export const OL_PLUGINS_CONFIG = 'olPluginsConfig';
 
 export interface IOLPluginsConfig {
 	Annotations: {
 		displayId: '0' | '1';
+	},
+	AnnotationsContextMenu: {
+		metadata: {
```

Comment:
why do yo add the "metadata" section? just put the active and attributes directly under AnnotationsContextMenu